### ISSUE TITLE
Please add exit-hooks

### ISSUE THREAD

**Issue by ailisp** [quicklisp/quicklisp-projects#1212]
exit-hooks provide a portable way to automatically call some user defined function when exit Common Lisp. Like Python’s atexit.

The project can be found here: https://github.com/ailisp/exit-hooks

License: BSD 2 clause

**Comment by guicho271828** [comment]
Hi, are you interested in merging your work with trivial-signal?

**Comment by ailisp** [comment]
@guicho271828 
Hi, I run your trivial-signal, it's more powerful and useful than exit-hooks. But I think exit-hooks is easier to learn for a simply run-on-exit, so i decide to leave it alone. Thanks for your suggestion. In fact, I wrote this small project because I didn't find trivial-signal (by searching quickdocs.org something about exit hook/quit... I got nothing). I will add suggestion on exit-hooks' readme to suggest advanced requirements to use trivial-signal instead.
